news/_posts/2022-01-13-new-theme-announce.md

```diff
@@ -0,0 +1,71 @@
+
+## How to try the new theme? 
+
+Clone it from the [theme repository](https://github.com/PrestaShop/theme-refacto), run `npm install`, `npm run build`, enable it from the back office and off you go!
+
+## Good base for theme developers
+
+This new theme features two different CSS packs. The first one is dedicated to the functional part, the second one is only for styling.
+
+Commenting out `@import "custom/_custom";` in theme.scss file will get you basically a pure Boostrap 5 theme base. You can get working immediately and not waste time by cleaning things you don't need.
+
+## When will it be released?
+
+Because of the complexity of this task, we decided not to announce a release date. This means that, depending on how much time people can spare, the first version of the theme could be released in PrestaShop v8, v9, or later.
+
+We hope to release an Alpha version of the new theme at the same time as the v8 release, but it could be after, even inside a minor version.
+
+This release date is not only dependent on the maintainers, but also on the many contributors who work hard to make this project happen. So we would like to thank all the contributors mentioned [here] (https://github.com/PrestaShop/theme-refacto/graphs/contributors) and those who participate in the issues and reviews on GitHub! Keep up the good work. This theme is also your theme, feel free to contribute and join the adventure. :-)
+
+## How does the new theme look?
+
+Have a look at the [mockups](https://www.figma.com/file/LfVl5leeSKcVUhSaYwhbtM/New-Theme), and [the list of open issues](https://github.com/PrestaShop/theme-refacto/issues).
+
+## What about the Classic theme?
+
+As this new theme is getting bigger and more complete, we must consider what will happen for the Classic Theme when this new theme is ready?
+
+We do not plan to maintain both themes simultaneously, so when the new theme rises, it will be necessary to decide how to stop the support of the Classic Theme. Today we have not yet decided how this will happen.
+
+Rest assured we will make sure that the end-of-life of Classic theme and the transition to this new theme will be made very smooth.
+
+## Your feedback is a gift
+
+We need your feedback on this project, don't hesitate to [join us on slack](https://www.prestashop-project.org/slack/) to discuss with us. We're actively asking a lot of things to the community to make sure the new theme will be something that meets your expectations and needs.
+
+Are there some places in the shop where you need to display your content and there is no hook provided, or is hard to use? Let us know through [Github](https://github.com/PrestaShop/PrestaShop/issues) or on Slack.
+
+You can also comment in this [thread](https://github.com/PrestaShop/theme-refacto/issues/2) if you want to tell us about any problems, challenges that you encountered using and developing with the classic theme.
```

Comment:
Problems encountered with new theme or classic theme?

Reply:
`us about any problems, challenges that you encountered using and developing with the classic theme.`|

developing with the classic theme